library/core/src/str/mod.rs

```diff
@@ -946,6 +952,12 @@ impl str {
     ///
     /// assert_eq!(None, iter.next());
     /// ```
+    ///
+    /// If the string is empty or all ASCII whitespace, the iterator yields no string slices:
+    /// ```
+    /// assert_eq!("".split_whitespace().next(), None);
+    /// assert_eq!("   ".split_whitespace().next(), None);
```

Comment:
This doc comment is attached to `split_ascii_whitespace` but is mentioning `split_whitespace`

```suggestion
    /// assert_eq!("".split_ascii_whitespace().next(), None);
    /// assert_eq!("   ".split_ascii_whitespace().next(), None);
```

Reply:
Nice catch. I've rejected the PR but will fix it manually if my rejection is too late.

Reply:
My bad.

Reply:
Updated...